src/main/java/io/phasetwo/keycloak/events/ScriptEventListenerProvider.java

```diff
@@ -66,6 +66,7 @@ public void onEvent(AdminEvent event, boolean b) {
             bindings -> {
               bindings.put("event", event);
               bindings.put("realm", Events.getRealm(session, event));
+              bindings.put("user", Events.getUser(session, event));
```

Comment:
Since we are not actually binding a "User" which is the subject of the action, but the authenticated user that is performing the action, can you please change it to `authUser` in the bindings?

Reply:
It is the same object/model thought isnt it, but the `authUser` would only be present on the AdminEvent which is only for authenticated users?